cmd/generate/runner.go

```diff
@@ -106,7 +106,7 @@ func (r *runner) run(ctx context.Context, cmd *cobra.Command, args []string) err
 	gen, err := generator.New(&generator.Config{
 		Fs:               store,
 		DecryptTraverser: decryptTraverser,
-		ProjectVersion:   project.AppControlPlaneVersion(),
+		ProjectVersion:   project.Version(),
```

Comment:
I changed that only to simplify the code. I don't think that it is important that the generated version is `0.0.0` when using CLI. I'm not even sure if it should be `0.0.0`.